ci-operator/jobs/openshift/oauth-server/openshift-oauth-server-master-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
```

Comment:
we should not promote the image while origin CI is promoting the same one, only after moving away from staging